03_gcc41 by Ben Hutchings <ben@decadentplace.org.uk>
Fix compile failure with GCC 4.1
closes: #355989
Index: ladspa-sdk/src/plugins/sine.cpp
===================================================================
--- ladspa-sdk.orig/src/plugins/sine.cpp	2009-06-05 14:04:43.000000000 +1000
+++ ladspa-sdk/src/plugins/sine.cpp	2009-06-05 14:06:50.000000000 +1000
@@ -68,6 +68,22 @@
 
 /*****************************************************************************/
 
+LADSPA_Handle instantiateSineOscillator(const LADSPA_Descriptor *,
+					unsigned long SampleRate);
+void connectPortToSineOscillator(LADSPA_Handle Instance,
+				 unsigned long Port,
+				 LADSPA_Data * DataLocation);
+void activateSineOscillator(void * pvHandle);
+void cleanupSineOscillator(void *pvHandle);
+void runSineOscillator_FreqAudio_AmpAudio(LADSPA_Handle Instance,
+					  unsigned long SampleCount);
+void runSineOscillator_FreqAudio_AmpCtrl(LADSPA_Handle Instance,
+					 unsigned long SampleCount);
+void runSineOscillator_FreqCtrl_AmpAudio(LADSPA_Handle Instance,
+					 unsigned long SampleCount);
+void runSineOscillator_FreqCtrl_AmpCtrl(LADSPA_Handle Instance,
+					unsigned long SampleCount);
+
 class SineOscillator {
 private:
 
